deploy/templates/notificationtemplates/stonesoup/userprovisioned/notification.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>
+    Notice: Your Stone Soup account is now provisioned.
+  </title>
+  <style>
+    a:hover {
+      text-decoration: underline !important;
+    }
+    p {
+      text-align: left;
+      margin: 30px 0;
+    }
+  </style>
+</head>
+
+<body
+        style="
+       padding: 10px;
+       padding: 0;
+       background-color: #f9f9f9;
+       font-family: 'Open Sans', sans-serif;
+       font-size: 15px;
+       font-weight: lighter;
+       line-height: 1.2;"
+>
+<div
+        style="
+       min-height: 300px;
+       max-width: 750px;
+       margin: 0 auto;
+       padding: 20px;
+       border: 1px solid #d7d7d7;
+       border-radius: 4px;
+       background-color: #fff;
+       box-shadow: 0 2px 4px #d7d7d7;"
+>
+
+  <p>
+    You are receiving this email because you have a stone soup account associated with {{.UserEmail}}.
+  </p>
+
+  <p>
+    Your account has been provisioned and is ready to use. Your account will be active for 30 days.
+    At the end of the active period, your access will be deactivated.
+  </p>
+
+  <p>
+    Please log in to {{.RegistrationURL}} to begin using your account.
+  </p>
+
+  <p>
+    Join the Dev Sandbox community and engage with the Red Hat team from the #dev-sandbox channel on DevNation slack workspace.
+    You can join using the following invite - https://dn.dev/DevNationSlack. You can also reach us via email at {{.ReplyTo}} with any questions.
+  </p>
+
+  <p>
+    Thanks,<br />
+    The Developer Sandbox for Red Hat OpenShift team
```

Comment:
Have you seen https://docs.google.com/document/d/11P8Gj8Q9gindbB9wmbFuIKv53SQL3b99vHUy0jemPfw/edit ?
Let's make the template as close to the proposed draft as possible. We will have to introduce more parameters like exact console link (instead of the reg-service URL) to make this template exactly as in the doc but we can do it in a different PR. For now let's just make it look very similar to what in the doc.

Reply:
added all four folders for appstudio here - [3df1eab](https://github.com/codeready-toolchain/host-operator/pull/754/commits/3df1eab38ec6e59824567e8d6c9d39bd6c43a3ff)